framework/src/base/MooseInit.C

```diff
@@ -21,6 +21,22 @@
 #include <omp.h>
 #endif
 
+#include <unistd.h>
+#include <signal.h>
+
+void
+my_sig_handler(int signum)
```

Comment:
I don't think we have a specific naming convention for free functions, since we really use them anywhere, but we probably should just follow our method naming convention: Camelcase
https://mooseframework.inl.gov/sqa/framework_scs.html